lib/kino_slack/message_interpolator.ex

```diff
@@ -0,0 +1,60 @@
+defmodule KinoSlack.MessageInterpolator do
+  def interpolate(message) do
+    ast = quote do: <<"">>
+    interpolate(message, ast)
```

Comment:
Instead of collecting the AST as a while, my recommendation is to collect the entries inside `:<<>>`. Something like this:

```suggestion
    reversed = interpolate(message, [])
    {:<<>>, [], reversed}
```

Where reversed is either {:"..", ..., ...} or strings in reverse order (as we accumulate them).

Reply:
Done.